pkg/repository/expenses_test.go

```diff
@@ -0,0 +1,66 @@
+package repository_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/monetr/monetr/pkg/internal/fixtures"
+	"github.com/monetr/monetr/pkg/internal/myownsanity"
+	"github.com/monetr/monetr/pkg/internal/testutils"
+	"github.com/monetr/monetr/pkg/models"
+	"github.com/monetr/monetr/pkg/repository"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRepositoryBase_GetSpendingById(t *testing.T) {
```

Comment:
Would also be good to add a test that shows we properly get the relation for requesting by funding schedule ID too.